build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.32"
+    ext.kotlin_version = "1.5.21"
```

Comment:
Are you using any new features from this version of Kotlin?

Reply:
Nope, most of the changes based on: https://kotlinlang.org/docs/releases.html#release-details are bug fixes or improvements on the Gradle loading.

Reply:
Okay. I remember running into some issues with Kotlin incompatibilities in some library that I was using and also some issues with having to upgrade the Gradle plugin, which then caused some other issue :(  Since there's nothing specific we need from Kotlin 1.5, let's just leave it at 1.4.32 which is what we're currently using in android-lantern.